mailpoet/lib/EmailEditor/Integrations/Core/Renderer/Blocks/AbstractBlockRenderer.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\EmailEditor\Integrations\Core\Renderer\Blocks;
+
+use MailPoet\EmailEditor\Engine\Renderer\ContentRenderer\BlockRenderer;
+use WP_Style_Engine;
+
+/**
+ * Shared functionality for block renderers.
+ */
+abstract class AbstractBlockRenderer implements BlockRenderer {
+  /**
+   * Wrapper for wp_style_engine_get_styles which ensures all values are returned.
+   *
+   * @param array $block_styles Array of block styles.
+   * @param bool $skip_convert_vars If true, --wp_preset--spacing--x type values will be left in the original var:preset:spacing:x format.
+   * @return array
+   */
+  protected function getStylesFromBlock(array $block_styles, $skip_convert_vars = false) {
```

Comment:
This method originally returned an object in the Column and Columns classes. Why did you decide to don't convert it again and use an array instead? I'm just curious.

Reply:
@lysyjan I think it was PHPStan, pre-commit hook, not liking the usage of stdClass. Rather than exclude everything it seemed easier to just leave in the original array format instead of casting.